azure/services/managedclusters/managedclusters.go

```diff
@@ -215,7 +228,21 @@ func (s *Service) Reconcile(ctx context.Context) error {
 			}
 		}
 
-		diff := cmp.Diff(propertiesNormalized, existingMCPropertiesNormalized)
+		clusterNormalized := &containerservice.ManagedCluster{
```

Comment:
do we actually need this normalization dance? we're comparing one user-provided field and another field which always has the same value (or we can enforce that)